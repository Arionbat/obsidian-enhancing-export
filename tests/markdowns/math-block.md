$$
\begin{align*}
\begin{rcases}
\lambda_{1}(a_{11}^{*}, \ldots, a_{22}^{*}) < 0 \\
\lambda_{2}(a_{11}^{*}, \ldots, a_{22}^{*}) < 0 \\
\end{rcases} & \Rightarrow \text{stable knot} \\

	\begin{rcases}
        \lambda_{1}(a_{11}^{*}, \ldots, a_{22}^{*}) > 0 \\
        \lambda_{2}(a_{11}^{*}, \ldots, a_{22}^{*}) < 0 \\
	\end{rcases} & \Rightarrow \text{saddle}
\end{align*}
$$