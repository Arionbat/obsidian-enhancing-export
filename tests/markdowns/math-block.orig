Pandoc
  Meta { unMeta = fromList [] }
  [ Para
      [ Str "$$"
      , SoftBreak
      , RawInline
          (Format "tex")
          "\\begin{align*}\n\\begin{rcases}\n\\lambda_{1}(a_{11}^{*}, \\ldots, a_{22}^{*}) < 0 \\\\\n\\lambda_{2}(a_{11}^{*}, \\ldots, a_{22}^{*}) < 0 \\\\\n\\end{rcases} & \\Rightarrow \\text{stable knot} \\\\\n\n    \\begin{rcases}\n        \\lambda_{1}(a_{11}^{*}, \\ldots, a_{22}^{*}) > 0 \\\\\n        \\lambda_{2}(a_{11}^{*}, \\ldots, a_{22}^{*}) < 0 \\\\\n    \\end{rcases} & \\Rightarrow \\text{saddle}\n\\end{align*}"
      , SoftBreak
      , Str "$$"
      ]
  ]
