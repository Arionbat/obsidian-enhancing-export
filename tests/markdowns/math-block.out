Pandoc
  Meta { unMeta = fromList [] }
  [ Para
      [ Math
          DisplayMath
          "\n\\begin{align*}\n\\begin{rcases}\n\\lambda_{1}(a_{11}^{*}, \\ldots, a_{22}^{*}) < 0 \\\\\n\\lambda_{2}(a_{11}^{*}, \\ldots, a_{22}^{*}) < 0 \\\\\n\\end{rcases} & \\Rightarrow \\text{stable knot} \\\\\n\n    \\begin{rcases}\n        \\lambda_{1}(a_{11}^{*}, \\ldots, a_{22}^{*}) > 0 \\\\\n        \\lambda_{2}(a_{11}^{*}, \\ldots, a_{22}^{*}) < 0 \\\\\n    \\end{rcases} & \\Rightarrow \\text{saddle}\n\\end{align*}\n"
      ]
  ]
